README.md

```diff
@@ -45,144 +52,86 @@ pip install eqlpy
-print(db_format)
-```
+from django.db import models
+from eqlpy.eqldjango import *
 
-#### EqlBool
+class Customer(models.Model):
+    name = EncryptedText(table="customers", column="name")
+```
 
-Handles boolean values.
+The above customer model would create a database table like this:
 
-```python
-eql_bool = EqlBool(True, 'users', 'is_active')
-db_format = eql_bool.to_db_format()
-print(db_format)
+```sql
+CREATE TABLE myapp_customer (
+    "id" bigint NOT NULL PRIMARY KEY GENERATED BY DEFAULT AS IDENTITY,
+    "name" cs_encrypted_v1 NOT NULL,
+);
 ```
 
-#### EqlDate
+[!NOTE]
+The column type in the database is `cs_encrypted_v1` which is encrypted type defined in [EQL](https://github.com/cipherstash/encrypt-query-language/blob/main/docs/reference/PAYLOAD.md). Internally EQL uses JSON.
 
-Handles date values.
+`eqlpy` supports many different data types, including numbers, dates and JSON.
+See the [Reference](#) docs for more information.
 
-```python
-from datetime import date
+TODO: Link to the reference docs.
 
-eql_date = EqlDate(date.today(), 'users', 'created_at')
-db_format = eql_date.to_db_format()
-print(db_format)
-```
 
-#### EqlFloat
+### Inserting data
 
-Handles floating-point values.
+`eqlpy` automatically encrypts values when saving to the database.
+You can interact with the Django model in the same way as you normally would.
 
-```python
-eql_float = EqlFloat(3.14, 'measurements', 'value')
-db_format = eql_float.to_db_format()
-print(db_format)
+```py
+>>> customer = Customer(name="Fred Flintstone")
+>>> customer.save()
 ```
 
-#### EqlText
-
-Handles text values.
+When retrieving a record, `eqlpy` automatically decrypts the value using the credentials defined in the setup step.
 
-```python
-eql_text = EqlText('Hello, World!', 'messages', 'content')
-db_format = eql_text.to_db_format()
-print(db_format)
+```py
+>>> customer = Customer.objects.get(customer.id)
+>>> customer.name
+'Fred Flintstone'
 ```
 
-#### EqlJsonb
+TODO: Show what is contained in the database at this stage.
 
-Handles JSONB values.
+### Queries
 
-```python
-eql_jsonb = EqlJsonb({'key': 'value'}, 'configs', 'data')
-db_format = eql_jsonb.to_db_format()
-print(db_format)
-```
+Queries work much the same way in `eqlpy` - even though the data is fully encrypted!
 
-### Parsing values from database format
+For example, to find a record with a given name:
 
-Use the `from_parsed_json` method to convert database-formatted JSON back to Python types.
+```py
+Customer.objects.get(name__eq="Alice Developer")
+```
 
-```python
-import json
+Text search works slightly differently.
+EQL doesn't define the `LIKE` operator, but instead uses an operator called `match`
+which works in a similar way but over encrypted values.
 
-db_data = '{"k": "pt", "p": "42", "i": {"t": "users", "c": "age"}, "v": 1, "q": null}'
-parsed = json.loads(db_data)
-value = EqlInt.from_parsed_json(parsed)
-print(value)  # Output: 42
+```py
+found = Customer.objects.get(name__contains="caro")
```

Comment:
```suggestion
found = Customer.objects.get(name__match="caro")
```
This has been [updated](https://github.com/cipherstash/eqlpy/pull/10/commits/b17be2e9d01a2e132697db32e731757d9e0d0601) in my branch